src/Data/Integer/Properties.agda

```diff
@@ -692,18 +787,180 @@ pos-distrib-* (suc x) (suc y) = refl
 *-monoʳ-≤-pos x (+≤+ {m = suc _} {n = suc _} m≤n) =
   +≤+ ((ℕₚ.*-mono-≤ m≤n (ℕₚ.≤-refl {x = suc x})))
 
+*-monoʳ-≤-non-neg : ∀ n → (_* + n) Preserves _≤_ ⟶ _≤_
+*-monoʳ-≤-non-neg (suc n) = *-monoʳ-≤-pos n
+*-monoʳ-≤-non-neg zero {i} {j} i≤j
+  rewrite *-zeroʳ i
+        | *-zeroʳ j
+        = +≤+ z≤n
+
+*-monoˡ-≤-non-neg : ∀ n → (+ n *_) Preserves _≤_ ⟶ _≤_
+*-monoˡ-≤-non-neg n {i} {j} i≤j
+  rewrite *-comm (+ n) i
+        | *-comm (+ n) j
+        = *-monoʳ-≤-non-neg n i≤j
+
+*-monoˡ-≤-pos : ∀ n → (+ suc n *_) Preserves _≤_ ⟶ _≤_
+*-monoˡ-≤-pos n = *-monoˡ-≤-non-neg (suc n)
```

Comment:
Do we need this as well as `*-monoˡ-≤-non-neg`?

Reply:
There are other `-pos` lemmas in the library which is why I was expecting
something like this to exist (and show up when I use `C-c C-z` with _≤_ and "pos") AFAICR.

It doesn't cost us anything to throw it in.